app/views/users/show.html.erb

```diff
@@ -1,2 +1,2 @@
 <h1><%= t(".welcome") %></h1>
-<p><%= t(".profile_information", email: user.email) %></p>
+<p><%= t(".profile_information", email: current_user.email, name: current_user.name, admin: current_user.admin) %></p>
```

Comment:
That is a lot of info to be passed into a translation.
I would change that to be simple with attributes and values for now.

Also, why pass the admin here? Most users will not be admin and all they will see is: `Admin: false`

Reply:
Whoops, sorry, really need to do a better show page here - I was using the show page info to test other things as it was handy to see the admin status there - obviously I forgot to remove it all!